agave/core/filters.py

```
@@ -21,3 +21,17 @@ def generic_query(query: QueryParams, excluded: list[str] = []) -> Q:
     return filters & Q(**fields)
+
+
+ignore_error_codes = [
+    'ValidationError',
+    'DoesNotExist',
+    'InvalidQueryError',
+]
+
+
+def can_ignore_error(data: dict) -> bool:
+    can_ignore_error = (
```

Ruff (0.8.2)

Local variable `can_ignore_error` is assigned to but never used

Remove assignment to unused variable `can_ignore_error`

(F841)